linkerd/docs/namer.md

```diff
@@ -105,21 +106,21 @@ linkerd provides support for service discovery via
 [Consul](https://www.consul.io/). Note that this support is still considered
 experimental.
 
-The Consul namer is configured with kind `io.l5d.experimental.consul`, and these parameters:
+The Consul namer is configured with kind `io.l5d.consul`, and these parameters:
 
 * *host* --  the Consul host. (default: localhost)
 * *port* --  the Consul port. (default: 8500)
```

Comment:
Should you mention the experimental parameter here, along with the host and port params?  Or maybe update the text on line 107 to say that the experimental param is required for this namer?